model/src/main/java/org/cbioportal/model/StructuralVariant.java

```diff
@@ -23,14 +23,11 @@
 
 package org.cbioportal.model;
 
-import java.io.Serializable;
 import org.cbioportal.model.UniqueKeyBase;
 
-public class StructuralVariant extends UniqueKeyBase implements Serializable{
+public class StructuralVariant extends UniqueKeyBase {
```

Comment:
maybe check if this should be in the redis cache (in that case it should be serializable)